UPGRADE-4.0.md

```diff
@@ -78,6 +78,28 @@ FrameworkBundle
  * The service `serializer.mapping.cache.apc` has been removed; use
    `serializer.mapping.cache.doctrine.apc` instead.
 
+ * The `framework.serializer.cache` option has been removed. Configure a cache pool
+   called `serializer` under `framework.cache.pools` instead.
+
+   Before:
+
+   ```yaml
+   framework:
+       serializer:
+           cache: serializer.mapping.cache.apc
+   ```
+
+   After:
+
+   ```yaml
+   framework:
+       serializer: ~
```

Comment:
I think this should be kept as the two code blocks aren't equal otherwise (the serializer is not enabled if this is not present).